spec/features/adventures/user_views_adventure_spec.rb

```diff
@@ -67,7 +92,33 @@
 
   scenario "authenticated user successfully switches an adventure from public" \
   " to private by clicking on the appropirate icon" do
+    user = FactoryGirl.create(:user)
+    visit new_user_session_path
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Log in'
+
+    bucket_list = FactoryGirl.create(:bucket_list, user_id: user.id)
+    adventure = FactoryGirl.create(
+      :adventure
+    )
+    bucket_list_adventure = FactoryGirl.create(
```

Comment:
Useless assignment to variable - `bucket_list_adventure`. Did you mean `bucket_list`?